governance.md

```diff
@@ -57,6 +57,22 @@ Qualification guidelines
 
 If you’d like to nominate yourself as an Approver, please create an [Issue in the community repository](https://github.com/spinnaker/community/issues). 
 
+### SIG Leads
+
+SIG Leads are responsible for management of a particular Special Interest Group within the project, managing day-to-day operations of their interest scope.
+
+Duties
+
+* Set agenda for, facilitate, and drive SIG meetings
+* Review and sponsor proposals related to the SIG
+* Triage inbound spinnaker/spinnaker issues related to the SIG
+* Review and merge or decline pull requests from the community
+
+Qualifications
+
+* Approver status
```

Comment:
What happens to current SIG leads who aren't currently approvers?

Reply:
Good question, my thoughts is that nothing changes who is holding positions today. I'm just trying to clarify responsibilities here, not shake things up.

Reply:
Are there any who are not approvers?  If so, why do we think that this should be a requirement?  If we agree that it should be a requirement, what are the non-approver SIG leads going to do to become approvers?

All that said, I am not clear I agree that it should be a requirement for all SIGs.  For example, what value does it bring to the community to require that the leads for the Summit SIG need to be approvers?  Maybe I am missing something about the scope of what being an "approver" is.

Reply:
The Docs SIG is a great example of one where the leads are not approvers (except of the docs repo) and probably aren't likely to ever become full approvers, but nor does that seem inappropriate, IMO.

Reply:
Security SIG is another example, as well.

Reply:
Very well, I'll just remove this from the qualifications. Definitely agree that Summit / Docs SIGs shouldn't need approvers. Security SIG, however, seems not like the others.

Reply:
"For SIGs that are primarily about technical issues, the SIG leads should be approvers"? That's still wishy-washy, so maybe that's just making it too complicated.

Reply:
We may want to include "Belongs to an organization with significant stake in project" as a qualification to being a SIG lead.